content/learn/book/getting-started/setup/_index.md

```diff
@@ -85,34 +85,37 @@ bevy = "0.5" # make sure this is the latest version
 ```
 ### Enable Fast Compiles (Optional)
 
-Bevy can be built just fine using default configuration on stable Rust. However for maximally fast iterative compiles, we recommend the following configuration:
+Bevy can be built just fine using default configuration on stable Rust. However there are a few things you can do to significantly improve iterative compile times:
 
-* **Enable Bevy's Dynamic Linking Feature**: This is the biggest iterative compilation time win and requires no special setup. When you have `bevy` as a dependency you can run your app with dynamic linking like this:
+* **Bevy's Dynamic Linking Feature**: This is the biggest iterative compilation time win and requires no special setup. When you have `bevy` as a dependency you can run your app with dynamic linking like this:
```

Comment:
This should probably also mention that this Feature doesn't always work on Windows.
At least it doesn't when you also use the `Zshare-generics` config: https://github.com/bevyengine/bevy/issues/1126#issuecomment-750000945

Reply:
Lets make sure we close #131 if you make this change.

Reply:
I think it probably makes sense to add this to the "configure cargo" section instead of to the "dynamic linking" section, just to keep the dynamic linking section simple.

Reply:
Thanks! I believe my recent changes resolves this.